apps/dev-portal-web/domains/docs/components/mdx.tsx

```diff
@@ -8,6 +8,7 @@ import { CardLink } from '@/domains/docs/components/CardLink'
 import { Grid } from '@/domains/docs/components/Grid'
 import { Link } from '@/domains/docs/components/Link'
 import { Tabs } from '@/domains/docs/components/Tabs'
+import { AmountDistributionCalculator } from '@/domains/docs/widgets/AmountDistributionCalculator'
```

Comment:
Widget is still a component 
```suggestion
import { AmountDistributionCalculator } from '@/domains/docs/components/widgets/AmountDistributionCalculator'
```

Reply:
okay